src/scene/mesh-instance.js

```diff
@@ -151,6 +151,27 @@ class MeshInstance {
         this.flipFaces = false;
     }
 
+    static _meshSet = new Set();
```

Comment:
Any reason not to make this a module global property? That way, it's truly private.

Reply:
good point, I guess from other languages I'm used to keeping stuff inside classes.
I'll try and see if syntax with # works? https://stackoverflow.com/questions/22156326/private-properties-in-javascript-es6-classes

Reply:
# syntax for private properties is very new in ECMA. It needs to be investigated how it compiles to both targets by Babel.

Reply:
this is what babel generates for both es5 and module build:

```
// helpers

var id$3 = 0;

function _classPrivateFieldLooseKey(name) {
  return "__private_" + id$3++ + "_" + name;
}

function _classPrivateFieldLooseBase(receiver, privateKey) {
  if (!Object.prototype.hasOwnProperty.call(receiver, privateKey)) {
    throw new TypeError("attempted to use private field on non-instance");
  }

  return receiver;
}

// variable declaration
var _meshSet = _classPrivateFieldLooseKey("meshSet");

// variable use
_classPrivateFieldLooseBase(MeshInstance, _meshSet)[_meshSet].add(mesh);

```

So that does not feel ideal for performance critical code because of the extra hasOwnProperty call on each access to it to validate access. For less performance critical code, this should be fine?

Reply:
Yeah. A bit ugly though. Personally, I would just make it a module global. 😆

Reply:
agreed and done.